edge_api/pom.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.7.5</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>com.example</groupId>
+    <artifactId>edge_api</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>edge_api</name>
+    <description>edge_api</description>
+    <properties>
+        <java.version>17</java.version>
```

Comment:
Add the sonarcloud properties. Although they will mostly be overwritten by the input from the workflows. (Look at the other microservices)